example/src/ListWithHeader/Header.tsx

```diff
@@ -16,7 +16,8 @@ import { COLORS } from '../common';
 const SIGNET = require('./signet.png');
 const TEXT = require('./text.png');
 
-export const HEADER_HEIGHT = Platform.OS === 'web' ? 64 : 192;
+export const HEADER_HEIGHT =
+  Platform.OS === 'web' ? 64 : Platform.OS === 'macos' ? 128 : 192;
```

Comment:
Can we make the header the same as on web? Images seem to always be stretched, it may make sense to set explicit sizes for them instead of relying on the platform to properly contain them.

Reply:
Sure, I'll open a separate PR for it to avoid cluttering this one any further.